exporter_test.go

```diff
@@ -1325,6 +1325,14 @@ func TestClusterSlave(t *testing.T) {
 			t.Errorf("Did not find key [%s] \nbody: %s", want, body)
 		}
 	}
+	for wantedKey, wantedVal := range map[string]int{
+		`master_host="192.168.48.8"`: 5,
```

Comment:
Can you shorten this to just `192.168.`? I _think_ there's no guarantee the container will always get the same IP but the subnet should be consistent.

Reply:
port changes too 2333

Reply:
Isn't the port fixed?

Reply:
Last time it was 7002, this time it was 7001, is the prefix 700?

Reply:
Ah, you're right, I guess the three hosts select a master and yeah, they ports all start with `700` --> https://github.com/oliver006/redis_exporter/blob/master/.drone.yml#L54

Reply:
`master_host="172.20.0.9"`  = =

Reply:
`172.20.` is fine, too. That'll be the IP range for docker in drone.